tests/test_nn/test_representation.py

```diff
@@ -13,13 +13,13 @@
 import pykeen.nn.node_piece
 import pykeen.nn.representation
 from pykeen.datasets import get_dataset
-from pykeen.triples.generation import generate_triples_factory
 from tests import cases, mocks
 
 try:
     import transformers
 except ImportError:
     transformers = None
+error = pykeen.nn.pyg.try_import()
```

Comment:
maybe it is also fine to have the code duplicated instead of introducing this strange helper function.

Reply:
Agreed, this is too wild